plugins/woocommerce-blocks/tests/e2e/utils/cli.ts

```diff
@@ -1,9 +1,17 @@
 /**
  * External dependencies
  */
-import { exec } from 'child_process';
+import { ExecException, exec } from 'child_process';
 
-export function cli( cmd, args = [] ) {
+export function cli(
+	cmd: string,
+	args = []
+): Promise< {
```

Comment:
I ended up not using this due to needing to manipulate stdin, but this type improvement could be left in. TS couldn't infer the return type properly which was annoying.